back/docs/authentication_api.md

```
@@ -128,7 +133,48 @@ Authorization: Bearer eyJhbGciOiJIUzI1NiJ9...
+```
+
+### 4. ログアウト
+
+ユーザーをログアウトします。
+
+- **エンドポイント**: `POST /api/v1/logout`
+- **認証**: 必要
+
+#### リクエストヘッダー
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)